src/index.ts

```diff
@@ -351,4 +351,72 @@ export function apply(ctx: Context, config: Config) {
     cmd._options.sampler.fallback = config.sampler
     cmd._options.sampler.type = Object.keys(config.type === 'sd-webui' ? sampler.sd : sampler.nai)
   }, { immediate: true })
+
+  const subcmd = ctx.intersect(() => config.type === 'sd-webui')
+    .command('novelai.upscale')
+    .shortcut('放大', { fuzzy: true })
+    .option('scale', '-s <scale:number>', { fallback: 2 })
+    .option('resolution', '-r <resolution>', { type: resolution })
+    .option('crop', '-c', { type: 'boolean', fallback: true })
+    .option('upscaler', '-u <upscaler>')
+    .option('upscaler2', '-p <upscaler2>')
+    .option('upscaler2visibility', '-v <upscaler2visibility:number>')
```

Comment:
I don't think it's good to take up many letters.

Reply:
No thought to make a new name, anyone suggests?